docs/production-deployment/cloud/nexus/pricing.mdx

```diff
@@ -22,3 +24,8 @@ No Action results for handling or retrying the Nexus Operation itself.
 However, while the retry of the Nexus Operation incurs no charge, any billable action initiated by the handler (such as an Activity) will be charged if it fails and is subsequently retried.
 
 See [Pricing](/cloud/pricing) for additional details.
+
+<RelatedReadContainer>
+  <RelatedReadItem path="/evaluate/nexus" text="Evaluate Nexus" archetype="evaluate" />
```

Comment:
```suggestion
To learn more, check out the [Evaluate Nexus](/evaluate/nexus) section in our documentation.
```
The second link directs you back to this section so it can be removed. Not sure if these links are needed in general for this section.

Reply:
fixed